server/service/client.go

```diff
@@ -557,7 +557,7 @@ func (c *Client) ApplyGroup(
 
 	// Apply "no team" software installers.  Running every time to support
 	// removal of software installers on an empty config.
-	if appconfig != nil && appconfig.License.IsPremium() {
+	if len(specs.Teams) == 0 && appconfig != nil && appconfig.License.IsPremium() {
```

Comment:
What happens if a user adds a single query with `fleetctl apply`? Will it wipe their No Team software?